### PR TITLE
Split ZHA "Recommended" from "Other Supported Adapters" in two sections

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Change listing of Silicon Labs and Texas Instruments Zigbee adapters to the top of the Known working Zigbee radio modules so that they are listed before deconz (ConBee/RaspBee) Zigbee adapters, reasons being that both Silabs and TI adapters have many more available options on the market as well as being better supported in ZHA/zigpy than deconz (ConBee/RaspBee) radios.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Lets just sort the list alphabetically.

**Comment by Hedda** [comment]
> Lets just sort the list alphabetically.

That would make it even worse from the suggested compatibility point-of-view since Digi XBee is the least / poorest supported radio by ZHA/zigpy developers today.

It is my belief that many new ZHA end-users will just buy the first adapter that is listed at the very top, so it would be better to have Silicon Labs listed at the top since the upcoming Home Assistant SkyConnect USB Stick (and Home Assistant Yellow) will be based on Silicon Labs radios.

**Comment by frenck** [comment]
It is a list of "Known working Zigbee radio modules".
The order should not matter. We should not make it a search game, lets's please sort the list alphabetically.

**Comment by puddly** [comment]
The technical reasons for why some some adapters are better-supported than others don't belong in user documentation but this PR is correctly ordering them.  I don't want to encourage people to buy anything but TI, SiLabs, or a Conbee (in that order) and expect a smooth experience with ZHA.

Perhaps we can instead split this into two sections: "Recommended Adapters" and a less-prominent "Other Supported Adapters" (or something similar)?

**Comment by frenck** [comment]
Yeah that is fine, but ordering all of these per recommendation will lead to discussions always. Making a first section on recommended adapters and a second one with "others" sounds perfectly fine. It also clearly communicates the recommendation (which currently is implicit)

**Comment by Hedda** [comment]
> Perhaps we can instead split this into two sections: "Recommended Adapters" and a less-prominent "Other Supported Adapters" (or something similar)?

That is by the way almost exactly what Zigbee2MQTT official documentation does, as they have compatible adapters sorted in seperate lists for ”Recommended”, ”Not recommended”, and ”Experimental” while under those still grouping the adapters per chip radio type that they are based on, see -> https://www.zigbee2mqtt.io/guide/adapters/

**Comment by Hedda** [comment]
>> Perhaps we can instead split this into two sections: "Recommended Adapters" and a less-prominent "Other Supported Adapters" (or something similar)?
>
> Yeah that is fine, but ordering all of these per recommendation will lead to discussions always. Making a first section on recommended adapters and a second one with "others" sounds perfectly fine. It also clearly communicates the recommendation (which currently is implicit)

OK I have now changed the PR as per the split feedback, splitting lists of ZHA-supported adapters into two segments for which adapters/modules are recommended and which are not recommended for new purchases/buyers, which again is also more similar to how split Zigbee2MQTT their lists of supported adapters -> https://www.zigbee2mqtt.io/information/supported_adapters.html

**Comment by Hedda** [comment]
Closing in favor of https://github.com/home-assistant/home-assistant.io/pull/24493

**Comment by Hedda** [comment]
Updated: Split ZHA Recommended Adapters from Other Supported Adapters into two sections, as per feedback from @puddly.

> The technical reasons for why some some adapters are better-supported than others don't belong in user documentation but this PR is correctly ordering them. I don't want to encourage people to buy anything but TI, SiLabs, or a Conbee (in that order) and expect a smooth experience with ZHA.
> 
> Perhaps we can instead split this into two sections: "Recommended Adapters" and a less-prominent "Other Supported Adapters" (or something similar)?

With this the "**_Recommended Zigbee radio adapters and modules_**" section with modern/newer hardware from Silicon Labs, Texas Instruments, and Dresden Elektronik, then an "**_Other support but not recommended Zigbee radio adapters or modules_**" section with legacy/older   hardware from Silicon Labs, Texas Instruments, and Dresden Elektronik, as well as Digi XBee and ZiGate.

**Comment by Hedda** [comment]
Oh, removed clarifying explanations on why some are recommended and others are not as not quite sure if that is a too long text?

### Recommended Zigbee radio adapters and modules

These adapters/modules listed below are currently recommended because they use powerful radio chips, can handle a large Zigbee network, use later firmware still maintained by their manufacturer, and are well-supported in this Zigbee implementation.

### Other support but not recommended Zigbee radio adapters or modules

These other radio adapters/modules listed below are also supported but generally not recommended for a new purchase because reasons like they by today's standards have either obsolete/underpowered chips, can not handle a large Zigbee network, use older firmware no longer maintained by their manufacturer, or just currently do not have many developers + testers that keep actively developing the support for them with this Zigbee implementation.

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -77,25 +77,33 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
...
-  - [Bitron Video/Smabit BV AV2010/10 USB-Stick](https://bv.smabit.eu/index.php/smart-home-produkte/zb-funkstick/) with Silicon Labs Ember 3587
-  - Telegesis ETRX357USB/ETRX357USB-LR/ETRX357USB-LRS+8M (Note! These first have to be [flashed with other EmberZNet firmware](https://github.com/walthowd/husbzb-firmware))
 - Texas Instruments based radios (via the [zigpy-znp](https://github.com/zigpy/zigpy-znp) library for zigpy)
   - [CC2652P/CC2652R/CC2652RB USB stick, module, or dev board hardware flashed with Z-Stack coordinator firmware](https://www.zigbee2mqtt.io/information/supported_adapters)
   - [CC1352P/CC1352R USB stick, module, or dev board hardware flashed with Z-Stack coordinator firmware](https://www.zigbee2mqtt.io/information/supported_adapters)
+- dresden elektronik deCONZ based Zigbee radios (via the [zigpy-deconz](https://github.com/zigpy/zigpy-deconz) library for zigpy)
+  - [ConBee II (a.k.a. ConBee 2) USB adapter from dresden elektronik](https://phoscon.de/conbee2)
+  - [RaspBee II (a.k.a. RaspBee 2) Raspberry Pi Shield from dresden elektronik](https://phoscon.de/raspbee2)
+
+### Other support but not recommended Zigbee radio adapters or modules
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
### Other supported but not recommended Zigbee radio adapters or modules
```

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -106,7 +114,7 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
   - [PiZiGate (ZiGate Raspberry Pi module)](https://zigate.fr/produit/pizigatev2/)
   - [ZiGate-Ethernet (Ethernet gateway board for PiZiGate)](https://zigate.fr/produit/zigate-ethernet/)
   - [ZiGate + WiFi Pack](https://zigate.fr/produit/zigatev2-pack-wifi/)
-
+
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion

```

**Review by frenck — APPROVED** [review]
Thanks, @Hedda 👍 

../Frenck

**Pull request merged**